packages/validator/src/lib/psi/PsiResult.ts

```diff
@@ -30,6 +30,20 @@ export type LighthouseCategories = {
   [key in LighthouseCategoryName]: LighthouseCategory;
 };
 
+export type LighthouseAuditDetailsItem = {
+  [key: string]: number;
+}
+
+export type LighthouseAudit = {
+  id: string;
+  title: string;
+  description: string;
+  details: {
+    type: string;
+    items: LighthouseAuditDetailsItem[];
```

Comment:
Could you add a comment that PSI returns a list here but we only ever use the first item?

Reply:
added